### PR TITLE
Feature / Implement API call to add to cart

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Include Cart mock handlers in msw.
Pass lineId to product page component.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

[GRW-1276]: https://hedvig.atlassian.net/browse/GRW-1276?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #313** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/313" target="_blank">Graphite</a> 
    * **PR #314** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/314" target="_blank">Graphite</a> 
      * **PR #315** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/315" target="_blank">Graphite</a> 
        * **PR #316** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/316" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/316?utm_source=stack-comment).

File: apps/store/src/mocks/CartMock/CartMock.ts [hunk 1]
```diff
@@ -18,4 +18,28 @@ export const mockCartHandlers = [
...
+      ctx.data({
+        cart: {
+          linesAdd: { cart: cart ? dbToAPI(cart) : null },
+        },
+      }),
+    )
+  }),
+
+  api.mutation(CART_LINES_REMOVE, (req, res, ctx) => {
+    const cart = cartLinesAdd(req.variables.shopSessionId, req.variables.lineId)
```
Comment (guilhermespopolin) [inline apps/store/src/mocks/CartMock/CartMock.ts:35]: Shouldn't this be calling a `cartLinesAdd` thing? 🤔

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/mocks/CartMock/CartMock.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/mocks/CartMock/CartMock.ts:35]: whoops, remove I guess

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/316?utm_source=stack-comment).


Job Reference: Ve8NsZQh9VQTlrlOXPrV

**Pull request merged**